deepcell/datasets/tissue_net.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2016-2023 The Van Valen Lab at the California Institute of
+# Technology (Caltech), with support from the Paul Allen Family Foundation,
+# Google, & National Institutes of Health (NIH) under Grant U24CA224309-01.
+# All rights reserved.
+#
+# Licensed under a modified Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.github.com/vanvalenlab/deepcell-tf/LICENSE
+#
+# The Work provided may be used for non-commercial academic purposes only.
+# For any other use of the Work, including commercial use, please contact:
+# vanvalenlab@gmail.com
+#
+# Neither the name of Caltech nor the names of its contributors may be used
+# to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from deepcell.datasets import SegmentationDataset
+
+TissueNet_v1_0 = SegmentationDataset(
+    path='tissuenet_v1_0',
+    url='',
+    file_hash='',
+    secure=True
+)
+
+TissueNet_v1_1 = SegmentationDataset(
+    path='tissuenet_v1_1',
+    url='',
+    file_hash='',
+    secure=True
+)
+
+
+class TissueNet(SegmentationDataset):
```

Comment:
I am leaning towards this option as opposed the instantiated classes above because we can include relevant information in the docstring.